README.md

```diff
@@ -2,15 +2,17 @@
 
 ## Workflow
 
-[Overview slides](overview.pdf)
+[Introduction slides](introduction.pdf)
 
 If you have never done a PR before, feel free to ask for help from any of the designated helpers! Our whole job today is to help YOU. 
 
 ### 0. Get ready
 
-1. `install.packages(c("usethis", "devtools"))`
+1. `install.packages("pak")`
 
-1. Set up your GitHub personal access token with [?browse_github_pat](https://usethis.r-lib.org/reference/browse_github_token.html).
+1. `pak::pak("usethis")`
```

Comment:
Should this be `pak::pak("devtools")`? Which will pull in usethis.

Reply:
Oops, yes.